decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -809,10 +809,10 @@
             skip "Commentable comments has no votes" unless commentable.comments_have_votes?
 
             visit current_path
-            expect(page).to have_selector("#comment_#{comments[0].id} > .comment__footer > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
-            page.find("#comment_#{comments[0].id} > .comment__footer > .comment__footer-grid .comment__votes .js-comment__votes--up").click
-            expect(page).to have_selector("#comment_#{comments[0].id} > .comment__footer > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /1/)
-            expect(page).to have_selector("#comment_#{comment_on_comment.id} > .comment__footer > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
+            expect(page).to have_selector("#comment_#{comments[0].id} > [data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
+            page.find("#comment_#{comments[0].id} > [data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up").click
+            expect(page).to have_selector("#comment_#{comments[0].id} > [data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /1/)
+            expect(page).to have_selector("#comment_#{comment_on_comment.id} > [data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
```

Comment:
For some reason the suggestion feature from GH broke, but lets add the `div` in the selector: 

```ruby
            expect(page).to have_selector("#comment_#{comments[0].id} > div[data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
            page.find("#comment_#{comments[0].id} > div[data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up").click
            expect(page).to have_selector("#comment_#{comments[0].id} > div[data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /1/)
            expect(page).to have_selector("#comment_#{comment_on_comment.id} > div[data-comment-footer] > .comment__footer-grid .comment__votes .js-comment__votes--up", text: /0/)
```

Reply:
Sadly still failing. I've changed also the the selector on ```decidim-comments/app/views/decidim/comments/votes/create.js.erb```to: 
```
(function() {
  var commentId = <%= comment.id.to_json %>;
  var upVotes = <%= comment.up_votes.count.to_json %>;
  var downVotes = <%= comment.down_votes.count.to_json %>;
  var $comment = $("#comment_" + commentId);
  var $votes = $("> div[data-comment-footer] > .comment__footer-grid .comment__votes", $comment);
  var $upVote = $(".js-comment__votes--up", $votes);
  var $downVote = $(".js-comment__votes--down", $votes);

  $("span", $upVote).text(upVotes);
  $("span", $downVote).text(downVotes);

  $upVote.removeClass("is-vote-selected is-vote-notselected").removeAttr("disabled");
  $downVote.removeClass("is-vote-selected is-vote-notselected").removeAttr("disabled");

<% if comment.up_voted_by?(current_user) %>
  $upVote.addClass("is-vote-selected");
  $downVote.addClass("is-vote-notselected").attr("disabled", "disabled");
<% elsif comment.down_voted_by?(current_user) %>
  $upVote.addClass("is-vote-notselected").attr("disabled", "disabled");
  $downVote.addClass("is-vote-selected");
<% end %>
}());
```

on line 6 ```div[data-comment-footer]``` but still not passing. 

I suspect this is due to the grid selector ```.comment__footer-grid``` on the same line.

What do you think @andreslucena ?